mathvariant/mathvariant.js

```diff
@@ -0,0 +1,243 @@
+    '\u0635': {'misol': '\u{1EE11}', 'minit': '\u{1EE31}', 'mtail': '\u{1EE51}', 'mstrc': '\u{1EE71}', 'mloop': '\u{1EE91}', 'Bbb': '\u{1EEB1}'},
+    '\u0636': {'misol': '\u{1EE19}', 'minit': '\u{1EE39}', 'mtail': '\u{1EE59}', 'mstrc': '\u{1EE79}', 'mloop': '\u{1EE99}', 'Bbb': '\u{1EEB9}'},
+    '\u0637': {'misol': '\u{1EE08}', 'mstrc': '\u{1EE68}', 'mloop': '\u{1EE88}', 'Bbb': '\u{1EEA8}'},
+    '\u0638': {'misol': '\u{1EE1A}', 'mstrc': '\u{1EE7A}', 'mloop': '\u{1EE9A}', 'Bbb': '\u{1EEBA}'},
+    '\u0639': {'misol': '\u{1EE0F}', 'minit': '\u{1EE2F}', 'mtail': '\u{1EE4F}', 'mstrc': '\u{1EE6F}', 'mloop': '\u{1EE8F}', 'Bbb': '\u{1EEAF}'},
+    '\u063A': {'misol': '\u{1EE1B}', 'minit': '\u{1EE3B}', 'mtail': '\u{1EE5B}', 'mstrc': '\u{1EE7B}', 'mloop': '\u{1EE9B}', 'Bbb': '\u{1EEBB}'},
+    '\u0641': {'misol': '\u{1EE10}', 'minit': '\u{1EE30}', 'mstrc': '\u{1EE70}', 'mloop': '\u{1EE90}', 'Bbb': '\u{1EEB0}'},
+    '\u0642': {'misol': '\u{1EE12}', 'minit': '\u{1EE32}', 'mtail': '\u{1EE52}', 'mstrc': '\u{1EE72}', 'mloop': '\u{1EE92}', 'Bbb': '\u{1EEB2}'},
+    '\u0643': {'misol': '\u{1EE0A}', 'minit': '\u{1EE2A}', 'mstrc': '\u{1EE6A}'},
+    '\u0644': {'misol': '\u{1EE0B}', 'minit': '\u{1EE2B}', 'mtail': '\u{1EE4B}', 'mloop': '\u{1EE8B}', 'Bbb': '\u{1EEAB}'},
+    '\u0645': {'misol': '\u{1EE0C}', 'minit': '\u{1EE2C}', 'mstrc': '\u{1EE6C}', 'mloop': '\u{1EE8C}', 'Bbb': '\u{1EEAC}'},
+    '\u0646': {'misol': '\u{1EE0D}', 'minit': '\u{1EE2D}', 'mtail': '\u{1EE4D}', 'mstrc': '\u{1EE6D}', 'mloop': '\u{1EE8D}', 'Bbb': '\u{1EEAD}'},
+    '\u0647': {'minit': '\u{1EE24}', 'mstrc': '\u{1EE64}', 'mloop': '\u{1EE84}'},
+    '\u0648': {'misol': '\u{1EE05}', 'mloop': '\u{1EE85}', 'Bbb': '\u{1EEA5}'},
+    '\u064A': {'misol': '\u{1EE09}', 'minit': '\u{1EE29}', 'mtail': '\u{1EE49}', 'mstrc': '\u{1EE69}', 'mloop': '\u{1EE89}', 'Bbb': '\u{1EEA9}'},
+    '\u066E': {'misol': '\u{1EE1C}', 'mstrc': '\u{1EE7C}',},
+    '\u066F': {'misol': '\u{1EE1F}', 'mtail': '\u{1EE5F}',},
+    '\u06A1': {'misol': '\u{1EE1E}', 'mstrc': '\u{1EE7E}',},
+    '\u06BA': {'misol': '\u{1EE1D}', 'mtail': '\u{1EE5D}',},
+};
+
+const convertMathvariant = (el) => {
+    // If an mi element has a mathvariant attribute other than 'normal' and
+    // its textContent is a character in mathFonts, change the textContent
+    // to the desired math style and remove the attribute.
+    let mathVariant = el.getAttribute('mathvariant')
+    if (!mathVariant || mathVariant == 'normal')
+        return
+
+    let mathStyle = mathvariants[mathVariant]
+    if (!mathStyle)
+        return
+
+    let text = el.textContent
+    let val = ''
+
+    for (let i = 0; i < text.length; i++) {
+        let ch = text[i]
+        if (ch in mathFonts && mathStyle in mathFonts[ch])
+            val += mathFonts[ch][mathStyle]
+        else
+            val += ch
+    }
+    el.innerHTML = val
+    el.removeAttribute('mathVariant')
```

Comment:
```suggestion
    el.removeAttribute('mathvariant')
```

the attribute name must be lower case here